as/plugins/org.jboss.ide.eclipse.as.ui/jbossui/org/jboss/ide/eclipse/as/ui/JBossServerUIPlugin.java

```diff
@@ -124,10 +124,6 @@ public static JBossServerUIPlugin getDefault() {
 	public ResourceBundle getResourceBundle() {
 		return resourceBundle;
 	}
-
-	public void earlyStartup() {
-		JBossServerCorePlugin.getDefault();
-	}
```

Comment:
Hey Denis:  Why are we removing this line here? Pretty sure we need as.core to start up when as.ui starts, since as.core adds listeners and various other things that need to be added.